README.md

```diff
@@ -122,7 +122,10 @@ See the [open issues](https://github.com/terraform-provider-minio/terraform-prov
 
 ## License
 
-Distributed under the Apache License. See [LICENSE](./LICENSE) for more information.
+This work is dual-licensed under [Apache 2.0](./LICENSE.Apache-2.0) and [AGPL v3](./LICENSE.AGPL-3.0).
+The source contained within this repository is Apache 2.0 licensed, however due to `mdadm` being AGPL 3.0, the combined work is distributed as AGPL 3.0.
+
+`SPDX-License-Identifier: Apache-2.0 OR AGPL-3.0`
```

Comment:
```suggestion
`SPDX-License-Identifier: Apache-2.0 OR AGPL-3.0-or-later`
```

To be the same as inside `LICENSE`, and to have the [correct specifier](https://spdx.org/licenses/AGPL-3.0-or-later.html).